src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Inclusion/InclusionBlock/InclusionBlock.cs

```diff
@@ -12,6 +12,8 @@ public class InclusionBlock : ContainerBlock
 
         public string IncludedFilePath { get; set; }
 
+        public object File { get; set; }
```

Comment:
nit: to tell apart from `IncludedFilePath` ?
```suggestion
        public object ResolvedFilePath { get; set; }
```